src/vcd/parse/scopes.rs

```diff
@@ -138,6 +140,7 @@ pub(super) fn parse_var<R: std::io::Read>(
                     .cloned()
                     .chain([full_signal_name])
                     .collect::<Vec<String>>(),
+                human_readable_size: size,
```

Comment:
For the ModelSim use case this parameter name is maybe not the most obvious. The only thing we know is that it contains an index or a range.

Not a big deal though.